com.github.taburineagle.NeewerLite-Python.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
Typically, you should maintain it in your repo.

Reply:
It the goal we have a upstream PR that shoud be merged soon i hope